pkg/gke/update.go

```diff
@@ -429,7 +430,7 @@ func UpdateNodePoolSize(
 		return NotChanged, nil
 	}
 
-	logrus.Infof("updating size of node pool [%s] on cluster [%s]", *nodePool.Name, config.Name)
+	logrus.Infof("updating size of node pool [%s] from %d to %d on cluster [%s (%s)]", *nodePool.Name, *upstreamNodePool.InitialNodeCount, *nodePool.InitialNodeCount, config.Spec.ClusterName, config.Name)
```

Comment:
it is wrong, NodePool names can't be different from upstream. Using %+v as formatting verb is really bad idea.

Reply:
>NodePool names can't be different from upstream.

I am not sure I understand what you are trying to say, can you elaborate?

>Using %+v as formatting verb is really bad idea.

Why do you say that? I did not want to use `%v` since in case of multiple bool for e.g., user would not know what the values mean, `%+v` is more verbose even if it takes up more space when printed.